utrechtteam_vegu2021.html

```diff
@@ -0,0 +1,265 @@
+<!DOCTYPE html>
```

Comment:
Why not simply name this file vegu21.html? Then the url that we can share is oceanparcels.org/vegu21 (so without the 'utrechtteam_')